src/observable/BoundCallbackObservable.ts

```diff
@@ -138,11 +138,19 @@ function dispatch<T>(state: { source: BoundCallbackObservable<T>, subscriber: Su
   self.add(subject.subscribe(subscriber));
 }
 
-function dispatchNext({ value, subject }) {
+interface DispatchNextArg<T> {
+  subject: AsyncSubject<T>;
+  value: T;
+}
+function dispatchNext<T>({ value, subject }: DispatchNextArg<T>) {
```

Comment:
it's also consistency suggestion - current codebase destructures inside of code - is there specific reason to do it in parameter?

```
function do(xxx : xxxType) {
const {value, subject} = xxx;
}
```